docs/README.md

```diff
@@ -161,11 +161,13 @@ file name as `report.yaml`.
 
 When you follow the [partner connect documentation][partners], you could see
 details about adding a PGP public key.  Adding PGP public key is optional.  But
-if you have added that, you should see your public key in the _OWNERS_ file under
-your chart directory within your organization directory.
+if you have added that, you should see your public key in the _OWNERS_ file
+under your chart directory within your organization directory.  The PGP public
+key is available in the `publicPgpKey` attribute.  The value of that attribute
+must be conforming to [ASCII armor format][ascii-armor].
```

Comment:
can we provide some link here showing how to generate key in such format? The current link is very academic.

Reply:
Replaced the link with a Red Hat article: https://www.redhat.com/sysadmin/creating-gpg-keypairs